src/hotspot/share/compiler/compilationPolicy.cpp

```diff
@@ -170,7 +170,7 @@ void CompilationPolicy::replay_training_at_init_impl(InstanceKlass* klass, TRAPS
   }
 }
 
-void CompilationPolicy::replay_training_at_init(InstanceKlass* klass, TRAPS) {
+void CompilationPolicy::replay_training_at_init(InstanceKlass* klass, JavaThread* THREAD) {
```

Comment:
Please rename `THREAD` to `current`. `THREAD` is still inherently part of the `TRAPS` mechanism.

Reply:
ok